### DISCUSSION TITLE
DXMT

### DISCUSSION THREAD

**Discussion by cyrilvoynov** [italomandara/CXPatcher#214] [Ideas]
Can we expect DXMT implementation in near future? Due to last @3Shain [report](https://github.com/3Shain/dxmt/discussions/19) it seems like Deadlock is working just fine. I hope DXMT can fix massive lagging spikes during intense scenes and make it finally playable.

**Comment by 3Shain** [comment]
Please wait for the DXMT release. It's still under active development thus there won't be any quality assurance. We still have loads of things to fine-tune!

**Comment by italomandara** [comment]
I was considering that for a long time but i wanted to wait for dxmt to mature
